scst-policy/configuring.md

```diff
@@ -237,43 +176,34 @@ deployment namespace.
 
 ### <a id="provide-pol-auth-secrets"></a> Provide secrets for authentication in your policy
 
-You can provide secrets for authentication as part of the name pattern policy configuration provided your use case meets the following conditions:
+You can provide secrets for authentication as part of the name pattern policy
+configuration provided your use case meets the following conditions:
 
 * Your images and signatures reside in a registry protected by authentication.
 
-* You do not have `imagePullSecrets` configured in your runnable resources or
-in the `ServiceAccount`s that your runnable resources use.
+* You do not have `signaturePullSecrets` configured in your runnable resources
```

Comment:
the runnable resource doesnt specify signaturePullSecrets. it is the clusterimagepolicy